### PR TITLE
Added a return to ui/zaken/<bronorganisatie>/<identificatie>/ url to …

### PR DESCRIPTION
**Pull request by damm89**
…get review request endpoint

### CODE REVIEW

File: backend/src/zac/contrib/kownsl/views.py [hunk 1]
```diff
@@ -113,7 +114,18 @@ def get(self, request, request_uuid):
...
+        zaak = get_zaak(zaak_url)
+        return_url = get_ui_url(
+            [
+                "ui",
+                "zaken",
+                zaak.bronorganisatie,
+                zaak.identificatie,
+            ]
+        )
+        data = {**review_request, "return_url": return_url}
```
Comment (sergei-maertens) [inline backend/src/zac/contrib/kownsl/views.py:127]: this is called `frontend_url` in the Kownsl schema, not `return_url`

```suggestion
        data["frontend_url"] = return_url
```

**Review by sergei-maertens — CHANGES_REQUESTED** [review]
I don't understand where those frontend package changes keep coming from? Can you maybe just not commit those?

File: backend/src/zac/contrib/kownsl/views.py (hunk 1 above)
Comment (damm89) [reply backend/src/zac/contrib/kownsl/views.py:127]: Are you sure?

From https://kownsl.utrechtproeftuin.nl/api/docs/:
frontend_url: The frontend URL for reviewers to submit their review.

Looking at https://github.com/GemeenteUtrecht/kownsl/blob/master/src/kownsl/api/serializers.py this seems to refer back to "core:detail" of kownsl. 

This is to go return to the zaak-detail page.

Comment (sergei-maertens) [reply backend/src/zac/contrib/kownsl/views.py:127]: in that case I am wrong - but I don't understand why the backend then has to provide that URL?

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**